demo/demo.html

```diff
@@ -3,11 +3,15 @@
   <head>
     <title>Demo of Logic and View Integration</title>
     <script src="../dist/components.js"></script>
-    <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script>
+    <script src='./temp/sfx-all-plugins-bundle.js'></script> 
+    <!-- above tag to be removed when the cdn PR passes with the updated bundle -->
```

Comment:
please note that this script tag is temporary to enable you all to run my demo  while I wait for my logic pr to pass and the CDN to be updated. Thanks!

Reply:
You might find `git add -p` to be useful in the future. It allows you to commit only parts of files.